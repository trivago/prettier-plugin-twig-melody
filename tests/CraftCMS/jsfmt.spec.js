run_spec(__dirname, ["melody"]);
