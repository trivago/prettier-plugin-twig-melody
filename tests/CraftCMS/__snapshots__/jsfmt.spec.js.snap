// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav.melody.twig 1`] = `
<ul id="nav">
    {% nav entry in entries %}
        <li>
            <a href="{{ entry.url }}">{{ entry.title }}</a>
            {% ifchildren %}
                <ul>
                    {% children %}
                </ul>
            {% endifchildren %}
        </li>
    {% endnav %}
</ul>

{%- nav entry in entries -%}
        
{%- endnav -%}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<ul id="nav">
    {% nav entry in entries %}
        <li>
            <a href="{{ entry.url }}">{{ entry.title }}</a>
            ifchildren
        </li>
    {% endnav %}
</ul>

{%- nav entry in entries -%}

{%- endnav -%}

`;
